test/controllers/api/tasks_controller_test.rb

```diff
@@ -76,6 +76,24 @@ class Api::TasksControllerTest < ActionController::TestCase
                      session: set_session_user(User.current)
           assert_response :not_found
         end
+
+        it 'shows duration column in exported task' do
```

Comment:
This applies to all the tests, hitting `show`, `index` or `sub_tasks` endpoints with a request does not do an export so mentioning "exported task" in the test name might be misleading